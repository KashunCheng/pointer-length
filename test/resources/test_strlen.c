//{"func": "test"}
#include "string.h"
int test(int b, char* a)
{
  int r = 0;
  for(int i = 0; i < b; i++)
  {
    r += a[i];
  }
  return r;
}

int main(int argc, char** argv) {
  char a[5] = {1, 2, 3, 4, 0};
  int b = strlen(a);
  int result = test(b, a);
  return result;
}