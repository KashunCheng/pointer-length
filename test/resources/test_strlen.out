IsCapacity: 0
LeqCapacity: 1
IsLength: 1
LeqLength: 1
